go/gas-oracle/gasprices/gas_price_updater.go

```diff
@@ -63,14 +62,23 @@ func (g *GasPriceUpdater) UpdateGasPrice() error {
 	if err != nil {
 		return err
 	}
-	if latestBlockNumber < uint64(g.epochStartBlockNumber) {
+	if latestBlockNumber < g.epochStartBlockNumber {
 		return errors.New("Latest block number less than the last epoch's block number")
 	}
+
```

Comment:
Sanity checking this logic here - I first thought that we should do `i <= latestBlockNumber` but that causes the existing unit tests to break. The unit tests work without a problem with the implemented logic. Need to double check that this is the intended behavior.